vars/stageScanForSonarqube.groovy

```diff
@@ -1,5 +1,19 @@
-def call(def context, def requireQualityGatePass = false) {
-  if (context.sonarQubeBranch == '*' || context.sonarQubeBranch == context.gitBranch) {
+def call(def context, def requireQualityGatePass = false, def longLivedBranches = []) {
+  def prAnalysisEnabled = context.analysePullRequestsWithSonarQube
+  // If the Jenkinsfile author did not specify explicitly, we'll default to
+  // configuration in /etc/sonarqube/config.json.
+  if (prAnalysisEnabled == null) {
+    try {
```

Comment:
Should there be an info in the log somewhere to tip users off about the existence of `analysePullRequestsWithSonarQube`?

Reply:
I actually hope that no-one will need/want to use it. It is only there to be able to control it explicitly even if the config is not present.